docs/en_us/platform_api/source/course_structure/course_structure.rst

```diff
@@ -0,0 +1,208 @@
+########################################
+EdX Platform Course Structure API Module
+########################################
+
+This page contains information on using the Platform Course Structure API to:
+
+* :ref:`Get a list of courses in the edX platform <Get a List of Courses>`
+
+* :ref:`Get details about a course <Get Course Details>`
+
+* :ref:`Get a course's structure, or blocks <Get the Course Structure>`
+
+* :ref:`Get a courses grading policy <Get the Course Grading Policy>`
+
+.. _Get a List of Courses:
+
+**************************
+Get a List of Courses
+**************************
+
+.. autoclass:: course_structure_api.v0.views.CourseList
+
+**Example response**
+
+.. code-block:: json
+
+    {
+        "count": 809, 
+        "next": "https://courses.stage.edx.org/api/course_structure/v0/courses/?page=3", 
```

Comment:
These URLs refer to `courses.stage.edx.org` and can refer directly to prod (e.g. `https://courses.edx.org/api/course_structure/v0/courses/?page=3`) instead.